ap.md

```diff
@@ -0,0 +1,256 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+# Build Your Own Internet
+</div>
+<div class="large-7 small-12 columns">
+
+As explained in [wifi tutorial](/wifi.html), Tessel 2 can easily connect to wifi networks using the command line tool. If we're using a device connected to the same network as the Tessel, then we can connect to it through our device's web browser. Like most network-connected devices, communication over the web depends on a shared router, or access point, to direct requests and responses between every device on the network or through the Internet. This is fine and dandy if we want to use Tessel in a place with an available network, but what happens when that's not possible? 
+
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/router.png)
+</div>
+<div class="large-7 small-12 columns">
+Instead of requiring a shared network through a standalone router, we can use the command line tool for configuring Tessel to emit a custom Wifi network and use it as an access point. Then, using Node, we can also start a web server for communicating with any connected device. Unfortunately, we won't be able to connect to the Internet but there is still a lot we can do without that. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/Tessel-AP.png)
+</div>
+</div>
+
+<div class="row">
+<div class="large-12 columns">
+
+<hr>
+In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
+
+`mkdir tessel-router`
+
+`cd tessel-router`
+
+`t2 init`
+
+Rename the “index.js” file you’ve just created to “ap.js”, then copy and paste the below script over the existing text:
+
+{% highlight javascript %}
+// Import the interface to Tessel hardware
+var tessel = require('tessel');
+// Load the http module to create an http server.
+var http = require('http');
+
+// Configure our HTTP server to respond with "Hello from Tessel!" to all requests.
+var server = http.createServer(function (request, response) {
+  response.writeHead(200, {"Content-Type": "text/plain"});
+  response.end("Hello from Tessel!\n");
+});
+
+// Listen on port 8080, IP defaults to 192.168.1.101. Also accessible through [tessel-name].local
+server.listen(8080);
+
+// Put a friendly message on the terminal
+console.log("Server running at http://192.168.1.101:8080/");
+
+{% endhighlight %}
+
+Now that our server is in place, let's get our access point set up. In the terminal, run the following command:
```

Comment:
I still think it would be better to clearly delineate the creation of the AP from the server-running part into Part 1 and Part 2. Specifically: Part 1 = "here is the command line to create an AP. Here is a GIF of you scanning for wireless networks and finding the one you just made. Bonus: try making it with a password (link to CLI docs)" Part 2 = everything else.

Does this sound reasonable to you?

I am concerned that people will not know what pieces of the tutorial pertain to the creation of an access point, and which pertain to the creation of a server (I usually operate under the assumption that people should be able to learn to program by using our Tessel tutorial, so won't recognize a Node server when they see one).